src/components/DrawerMenu.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <v-navigation-drawer v-model="drawer" app clipped>
```

Comment:
v-model open up the variable fo rmodification, so you should look into the API for v-navigation-drawer to see how to do this without two-way binding.
The error I get is:
```
[Vue warn]: Avoid mutating a prop directly since the value will be overwritten whenever the parent component re-renders. Instead, use a data or computed property based on the prop's value. Prop being mutated: "drawer"

found in

---> <DrawerMenu> at src/components/DrawerMenu.vue
       <VApp>... (1 recursive calls)
         <App> at src/App.vue
           <Root>
```

Reply:
I still have the issue and tried everything :/